src/templates/discordTemplate.js

```diff
@@ -0,0 +1,2 @@
+import createRedirect from "../components/redirect"
+export default createRedirect("https://discord.gg/pxrBmy4")
```

Comment:
do we even need one file per each? can't we call createRedirect directly on gatsby-node.js ?

Reply:
Gatsby :(